proctor-consumer/src/test/java/com/indeed/proctor/consumer/spring/TestProctorConsumerUtils.java

```diff
@@ -228,15 +228,11 @@ public void testParseForceGroupsList() {
         assertThat(parseForceGroupsList("fasdfasdf")).isEmpty();
         assertThat(parseForceGroupsList("test" + Integer.MAX_VALUE + "0")).isEmpty();
         assertThat(parseForceGroupsList("test-")).isEmpty();
-        assertThat(parseForceGroupsList("test0-")).isEmpty();
         //Test single group
         assertThat(parseForceGroupsList("somerandomtst1"))
                 .hasSize(1)
                 .containsEntry("somerandomtst", 1);
         // not sure if this case needs to be supported...
-        assertThat(parseForceGroupsList("somerandomtst*  1"))
```

Comment:
I don't believe this should be a valid force group parameter.

Reply:
Yeah, this seems invalid to me too.